package.json

```diff
@@ -30,12 +30,18 @@
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
+  ],
+  "types": "types/index.d.ts",
+  "ignores": [
+    "types"
   ],
```

Comment:
"files" is an include list, so an exclude list like "ignores" probably isn't needed
```suggestion
```